integrations/mediawiki.js

```diff
@@ -0,0 +1,178 @@
+const util = require('util');
+const fetch = require('node-fetch');
+const groupArray = require('group-array');
+const cron = require('node-cron');
+
+module.exports = async function(robot, kredits) {
+
+  function messageRoom(message) {
+    robot.messageRoom(process.env.KREDITS_ROOM, message);
+  }
+
+  robot.logger.debug('[hubot-kredits] Loading MediaWiki integration...')
+
+  const Contributor = kredits.Contributor;
+  const Operator = kredits.Operator;
+
+  const apiURL =  process.env.KREDITS_MEDIAWIKI_URL + 'api.php';
+
+  function getContributorByWikiUser(username) {
+    return Contributor.all().then(contributors => {
+      let contrib = contributors.find(c => {
+        if (typeof c.accounts !== 'object') { return false; }
+        return c.accounts.find(a => {
+          a.url === `${process.env.KREDITS_MEDIAWIKI_URL}User:${username}`;
```

Comment:
do we need to do a toLowerCase or something to compare the usernames?

Reply:
Ideally we have the correct usernames on the blockchain. I haven't looked into the MediaWiki specifics there.

Reply:
we only have the usernames in the IPFS profile. which actually is fine because we want to store as little as possible in the contract